patches/.vpython3.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/.vpython3 b/.vpython3
+index 21e15085072f03737725b10588369eb4c2b620c6..2fc76f02b61fc8bc7f62df9e9fd3c62385b7cc8b 100644
+--- a/.vpython3
++++ b/.vpython3
+@@ -59,6 +59,17 @@ wheel: <
+ wheel: <
+   name: "infra/python/wheels/numpy/${vpython_platform}"
+   version: "version:1.20.3"
++  # A newer version of numpy is required on ARM64, but it breaks older OS versions.
```

Comment:
this is the exact upstream change, correct? So it will go away on its own?

Reply:
yes

Reply:
see what you get if you search for `ia32`